README.md

```diff
@@ -578,7 +578,10 @@ Returns true for [Forkables](#type-signatures) and false for everything else.
 #### `do :: (() -> Iterator) -> Future a b`
 
 A specialized version of [fantasy-do][19] which works only for Futures, but has
-the advantage of type-checking and not having to pass `Future.of`.
+the advantage of type-checking and not having to pass `Future.of`. Another
+advantge is that the returned Future can be forked multiple times, as opposed
+to with a general `fantasy-do` solution, where forking the Future a second time
+would introduce erronous behaviour.
```

Comment:
It's not only unpredictable, it's actually just wrong. Sometimes it even leads to reference errors and stuff like that.